Services/DataStructures/Instance.cs

```diff
@@ -15,16 +15,16 @@ public interface IInstance
         void InitComplete();
 
         // Fail with exception if the instance has already been initialized
-        void InitOnce([CallerMemberName]
-            string callerName = "", [CallerFilePath]
-            string filePath = "", [CallerLineNumber]
-            int lineNumber = 0);
+        void InitOnce(
+            [CallerMemberName] string callerName = "", 
```

Comment:
could you apply the same style also in the implementation below?

Reply:
Fixed.